workspaces/topology/plugins/topology/report.api.md

```diff
@@ -4,8 +4,14 @@
 
 ```ts
 import { BackstagePlugin } from '@backstage/core-plugin-api';
+import type { ComponentType } from 'react';
 import { JSX as JSX_2 } from 'react/jsx-runtime';
 import { RouteRef } from '@backstage/core-plugin-api';
+import { TranslationRef } from '@backstage/core-plugin-api/alpha';
+import { TranslationResource } from '@backstage/core-plugin-api/alpha';
+
+// @public
+export type Messages = typeof topologyTranslationRef.T;
```

Comment:
Do we really need this type exported in index.ts? I believe its not needed for an external usage of this plugin, or?